python/ray/experimental/placement_group.py

```diff
@@ -0,0 +1,23 @@
+import ray
+
+
+def placement_group(bundles: List[Dict[str, float]], strategy: str = "PACK",
```

Comment:
Define an enum for strategy.

Reply:
We can define a `PlacementGroup` type in this file. The advantages are:
1) avoid exposing placement group id (which is an implementation detail) to users. 
2) Document what a placement group is there.

Reply:
It's more Pythonic to use a string choice here I think.